RecogLib-iOS/SDK/Controllers/DocumentController.swift

```diff
@@ -147,25 +187,30 @@ public final class DocumentController: BaseController<DocumentResult>, DocumentC
     }
 
     deinit {
-        verifier.endHologramVerification()
+        if baseConfig.dataType == .video {
+            verifier.endHologramVerification()
+        }
         verifier.reset()
         resetDocumentVerifier()
     }
 
     public func configure(configuration: DocumentControllerConfiguration = .default) throws {
-        verifier.reset()
         verifier.endHologramVerification()
+        verifier.reset()
```

Comment:
Do we need to run this in a background thread, now that Reset is blocking? No need to change it in the PR and then retest just for this, just a thought for the future.